### ISSUE TITLE
Neovim is Incorrectly Configured on Windows Installations

### ISSUE THREAD

**Issue by Kaweees** [neovim/neovim#20127]
### Neovim version (nvim -v)

v0.7.2

### Vim (not Nvim) behaves the same?

Problem specific to neovim

### Operating system/version

Windows 10 10.0.19044

### Terminal name/version

Powershell 5.1

### $TERM environment variable

N/A

### Installation

CLI via Chocolatey

### How to reproduce the issue

Install Neovim via Chocolatey:
`choco install neovim -y`

### Expected behavior

According to the documentation, the Neovim configuration files should be found at:
`~/AppData/Local/nvim/init.vim`

### Actual behavior

However, upon running:
`:help nvim-configuration`
on Neovim, we see that Neovim has set the directory of the configuration files to be at
``~/AppData/Local/nvim-data/init.vim`
when in reality it is found at
`~/AppData/Local/nvim-data/init.vim`

Thank you for your time and patience

**Comment by zeertzjq** [comment]
> `:help nvim-configuration`

This help tag doesn't exist.

**Comment by Kaweees** [comment]
My apologies, I think I ran this command instead
`:echo stdpath('config')`

**Comment by justinmk** [comment]
is this a duplicate of https://github.com/neovim/neovim/issues/18349 ?

**Comment by 3N4N** [comment]
@Kaweees is your bug report correct?  For me, `init.vim` exists, is found and used at "~/AppData/Local/nvim".  You can get the location of config directory, as you already said, with `:echo stdpath('config')`.  For me the output is "~/AppData/Local/nvim".  I suspect you ran the command `echo stdpath('data')`, whose value indeed is `~/AppData/Local/nvim-data`, which is what's expected.

TL;DR:  I can't reproduce the bug.
